CgfConverter/Services/ArgsHandler.cs

```diff
@@ -24,9 +24,14 @@ public class ArgsHandler
         /// Directory to render to
         /// </summary>
         public string OutputDir { get; internal set; }
+
         /// <summary>
         /// Allows naming conflicts for mtl file
         /// </summary>
+        public LogLevelEnum LogLevel { get; set; } = LogLevelEnum.Critical;
```

Comment:
This is a good addition to the allowed args, but the comment attached to it is mixed up with the AllowConflicts one. :)  Can you swap those around?

Reply:
good catch